tool/tsh/gcp_test.go

```diff
@@ -258,3 +259,71 @@ func TestSortedGCPServiceAccounts(t *testing.T) {
 		})
 	}
 }
+
+func Test_gcpApp_Config(t *testing.T) {
```

Comment:
```suggestion
func Test_GCPApp_Config(t *testing.T) {
```

Reply:
I think the canonical name of the test should be `Test_{typeName}_{functionName}`? The type name is `gcpApp` and not `GCPApp`, as the type is not exported.

Reply:
Yea, I thought that I canceled this comment :-) I'm making it as resolved.